src/app/SidebarCategories.ts

```diff
@@ -62,6 +77,7 @@ const misc: SideBarCategory = {
 // Dashboard specific routes
 export const sideBarRoutes = [
 	jp,
+	person_summary,
 	admin,
 	misc,
```

Comment:
@yoshirahh FYI misc needs to stay in.  There's at least one page that exists only as an embed in the old FO app (manage staggered order).  For now it's routable because its in the misc tree and misc is routable.  I'll put something to render in the sidebar under Misc so it looks less stupid